app/main.py

```diff
@@ -14,4 +14,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return sum(1 for i in phrase if i.lower() == letter.lower())
```

Comment:
Use method `count` of string instead of `sum`

Reply:
Ready